hack/lib/init.sh

```diff
@@ -23,6 +23,7 @@ network/v1 \
 oauth/v1 \
 openshiftcontrolplane/v1 \
 operator/v1 \
+operatorcontrolplane/v1alpha1 \
```

Comment:
what is the rational of this group name?

Reply:
See https://github.com/openshift/enhancements/pull/289#issuecomment-620189204

Reply:
operator.openshift.io as suggest by @deads2k is fine. Who says this will only be used in control plane operator?